src/pages/Preferences.vue

```diff
@@ -19,8 +20,10 @@ export default Vue.extend({
   },
   layout: 'preferences',
   data() {
+    const settings: Settings = load();
```

Comment:
We're trying to ensure that all our communication between renderer an main is done via the http server so we'll want to avoid loading settings in this way. We've also already loaded preferences in the `fetch` hook below and we should be able to make use of that to accomplish our task 🙂.

We should be able to make use of the `getPreferences` getter if we want to work with the `currentNavItem`.